content/en/docs/aws/deploy/install-kubeflow.md

```diff
@@ -193,7 +206,7 @@ Traditional way to attach IAM policies to node group role is still working, feel
 
 ## Access Kubeflow central dashboard
 
-If you are using {{% config-uri-aws-cognito %}}, run following command to get Kubeflow service endpoint host name and copy link in browser.
+If you are using {{% kf-latest-version %}} config file, run following command to get Kubeflow service endpoint host name and copy link in browser.
```

Comment:
What's the value difference between `kf-latest-version` and `config-uri-aws-standard`?

Reply:
Currently, `kf-latest-version` should be `1.0.2`, and `config-uri-aws-standard` should be `https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_aws_cognito.v1.0.2.yaml`.

After Basic Authentication PR merged, no matter we use `https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_aws_cognito.v1.0.2.yaml` or `https://raw.githubusercontent.com/kubeflow/manifests/v1.0-branch/kfdef/kfctl_aws.v1.0.2.yaml`, we can get the ingress external-ip for accessing dashboard. 

So that's why I made the change.

Reply:
I see. I think only 1.0.2 doesn't have public endpoint, for all previous version, we use exact same way.